efi/secureboot_policy_test.go

```diff
@@ -340,43 +130,49 @@ func TestComputeDbUpdate(t *testing.T) {
 			update:        "testdata/updates1/dbx/MS-2016-08-08.bin",
 			quirkMode:     SigDbUpdateQuirkModeNone,
 			sha1hash:      testutil.DecodeHexStringT(t, "96f7dc104ee34a0ce8425aac20f29e2b2aba9d7e"),
-			newSignatures: 77,
+			newESLs:       1,
+			newSignatures: []int{77},
 		},
 		{
 			desc:          "AppendMsDbxUpdate/2",
 			orig:          "testdata/efivars2/dbx-d719b2cb-3d3a-4596-a3bc-dad00e67656f",
 			update:        "testdata/updates1/dbx/MS-2016-08-08.bin",
 			quirkMode:     SigDbUpdateQuirkModeDedupIgnoresOwner,
 			sha1hash:      testutil.DecodeHexStringT(t, "96f7dc104ee34a0ce8425aac20f29e2b2aba9d7e"),
-			newSignatures: 77,
+			newESLs:       1,
```

Comment:
does it make sense to have tests where this is > 1 ?

Reply:
I've added the 2020 dbx update now, which has more than 1 ESL.